README.md

```diff
@@ -0,0 +1,72 @@
+# elasticache
+
+Provides an ElastiCache Replication Group resource.
+
+Example Usage
+----------------
+
+```hcl
+module "elasticache" {
+  source = "git@github.com:techservicesillinois/terraform-aws-elasticache"
+
+  cluster_name       = "service-name"
```

Comment:
While I prefer qualified names like `cluster_name` more than David does, we've tended to standardize on using `name` to clarify what is the primary resource created by this module. Following this convention, the `cluster_name` variable should probably simply be `name`.

Reply:
done.